sdk/core/core-rest-pipeline/vitest.browser.config.ts

```diff
@@ -17,7 +24,12 @@ export default defineConfig({
       toFake: ["setTimeout", "Date"],
     },
     watch: false,
-    include: ["test/**/*.spec.ts"],
-    exclude: ["test/**/node/*.spec.ts"],
+    include: ["./dist-test/browser/**/*.spec.js"],
```

Comment:
what's the motivation behind changing this? Is it because we don't have the browser mapping anymore to translate the replacement files? Is there another way we can do that without having to rebuild a browser bundle every time?

I would be sad to regress the development flow here since not having to worry about re-building the bundle sped up my local debugging a lot

Reply:
It's a question of whether we have vitest build it for us versus building it for ourselves which we are doing in this case.